play/apps/tournament/views/tournament_bracket.py

```diff
@@ -61,8 +61,42 @@ def show_current_game(request, id):
 @login_required
 def show(request, id):
     tournament_bracket = TournamentBracket.objects.get(id=id)
+
+    def num_snakes_and_games(total_snakes):
+        import math
+        game_count = math.ceil(total_snakes/8)
+        snakes_per_game = total_snakes/game_count
+        min_snakes_per_game = math.floor(snakes_per_game)
+        max_snakes_per_game = math.ceil(snakes_per_game)
+        snakes_advancing = game_count*2
+        return game_count, (min_snakes_per_game, max_snakes_per_game), snakes_advancing
```

Comment:
I think this return is pretty horrendous, and going to be a maintenance burden.

Reply:
I agree, this is hard to parse here and where it's used.

Reply:
> @joram does this PR replace #74?

it does not, I closed the other one and will redoing after the refactor